deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -41,11 +41,11 @@
 >
 
     <p>
-        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for 12 hours and has been automatically idled.
+        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for long and has been automatically idled.
```

Comment:
```suggestion
        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for an extended period and has been automatically idled.
```